### ISSUE TITLE
required ruby version for ruby-oci 2.2.0

### ISSUE THREAD

**Issue by yahonda** [kubo/ruby-oci8#96]
Thanks for releasing the new version of ruby-oci8 2.2.0
According to NEWS update it drops ruby 1.8 support. I think it is a good news.

```
### Drop ruby 1.8 support

Use ruby-oci8 2.1.8 for ruby 1.8.
```
https://rubygems.org/gems/ruby-oci8/versions/2.2.0 still shows REQUIRED RUBY VERSION: as >= 1.8.0

I do not know the current required ruby version - 1.9, 2.0, 2.1 or 2.2. then did not open a pull request, just reporting .

**Comment by carld** [comment]
For those supporting ruby 1.8.7 code, can we use ruby-oci8 2.1.7

VERSION 2.1.8 and other changes (bytesize method ) happened here:
 
https://github.com/kubo/ruby-oci8/commit/782fb3fd565a49d117578c00f72bd32753c6ba15

That commit introduces a message in extconf.rb that suggests using 2.2.0

```
if RUBY_VERSION < "1.9"
  puts "Ruby-oci8 doesn't work ruby 1.8 since ruby-oci8 2.2.0."
  exit 1
end
```

Should the message say 2.1.7?

**Comment by kubo** [comment]
@yahonda 
Thank you!
I verified that `gem install ruby-oci8` failed on ruby 1.8.

I'll yank 2.2.0 by `gem yank` and release 2.2.0.1.
Ruby-oci8 2.2.0 has another problem in addition to this issue.
I supposed that [docs in rubydoc.info](http://www.rubydoc.info/gems/ruby-oci8/) was updated to 2.2.0 by
the ruby-oci8 release. However it is 2.1.8 even now.
I committed 2c50f33a526bebaa05032ee5729ebc03eecdc193 and changed URLs displayed on installation failure
to refer to docs generated from the github repository.

@carld 
Ruby-oci8 2.1.8 works with ruby 1.8.
782fb3fd565a49d117578c00f72bd32753c6ba15 is the next commit of ruby-oci8 2.1.8 release.
